pkgs/tools/misc/fselect/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fselect";
+  version = "0.6.6";
+
+  src = fetchFromGitHub {
+    owner = "jhspetersson";
+    repo = "fselect";
+    rev = version;
+    sha256 = "0lwh0qcyy9hmkxijlhp0inznrb0ibw1wcnnlqfxc71y4ar42f165";
+  };
+
+  cargoSha256 = "0s7yld3cwmyfdsqwcipc2l07xx5qn3knjmh4bbnjmnxiygaiszx9";
+
+  cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
is it possible to ask upstream to make a new release? the patch file is rather large and probably the result of upstream not checking in the latest version of it.